packages/components/psammead-most-read/src/testHelpers/doubleDigitOverride.js

```diff
@@ -1,34 +1,34 @@
 // These default measurements are to be used for 2nd column minimum width.
+// Services which use this default:
+// afaanoromoo, afrique, amharic, gahuza, hausa, hindi, igbo, indonesia,
+// kyrgyz, serbian, somali, swahili, ukchina, zhongwen (14)
 export const doubleDigitDefault = {
   group0: '2rem',
```

Comment:
Could you replace some of these `rem`s with existing Psammead spacing values? https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/spacings.js

Reply:
I'm not sure if it's worth replacing the values in this override file because it might look odd with half the values being in the psammead spacings and half very specific. I've left it as it is currently because I think it might be more readable with all rem values but I'd like some more thoughts on this.

Reply:
These values seem so specific to keeping things exactly aligned (I'm assuming based on trial and error?) that I think we should keep this as is, and then take another look at a solution that doesn't require this overrides file at all

Reply:
(In another PR, once we've released this)

Reply:
That sounds sensible 👍